### PR TITLE
Import ModuleExtension under try-except in OVC

### PR DESCRIPTION
**Pull request by mvafin**
### Details:
 - *Import ModuleExtension under try-except*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: tools/ovc/openvino/tools/ovc/convert_impl.py [hunk 1]
```diff
@@ -174,7 +173,11 @@ def prepare_ir(argv: argparse.Namespace):
         moc_front_end.add_extension(TelemetryExtension("ovc", t.send_event, t.send_error, t.send_stack_trace))
         if any_extensions_used(argv):
             for extension in argv.extension:
-                if not isinstance(extension, ModuleExtension):
+                try:
+                    from openvino.frontend.pytorch.module_extension import ModuleExtension
+                    if not isinstance(extension, ModuleExtension):
+                        moc_front_end.add_extension(extension)
```
Comment (slyalin) [inline tools/ovc/openvino/tools/ovc/convert_impl.py:179]: If PT FE is not available, then the import fails and no extensions will be registered for other FEs.

Comment (slyalin) [inline tools/ovc/openvino/tools/ovc/convert_impl.py:179]: Can we filter out `ModuleExtension` in a separate function without trying to import `ModuleExtension` in the loop. Just for a better structure in the code.

**Review by rkazants — APPROVED** [review]

**Comment by rkazants** [comment]
@mryzhov, does it solve a problem with tests in OV Tokenizers repo?

File: tools/ovc/openvino/tools/ovc/convert_impl.py (hunk 1 above)
Comment (mvafin) [reply tools/ovc/openvino/tools/ovc/convert_impl.py]: Done

**Comment by mryzhov** [comment]
> @mryzhov, does it solve a problem with tests in OV Tokenizers repo?
Yes, the problem has gone with this PR - https://github.com/openvinotoolkit/openvino_tokenizers/pull/89

**Review by slyalin — APPROVED** [review]
I have confirmed that it works with vLLM integration as before.

**Pull request merged**